### PR TITLE
add run time max value for  downstream max connections

### PR DESCRIPTION
**Pull request by ramaraochavali**
Fixes https://github.com/istio/istio/issues/29844

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[X ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by ramaraochavali** [comment]
@PiotrSikora Can you PTAL when you get chance?

**Comment by ramaraochavali** [comment]
@PiotrSikora gentle ping. Can you PTAL when you get chance?

File: tools/packaging/common/envoy_bootstrap.json [hunk 1]
```diff
@@ -31,6 +31,12 @@
                   "re2.max_program_size.error_level": 1024
               }
           },
+          {
+            "name": "global config",
+            "static_layer": {
+                "overload.global_downstream_max_connections": 2147483647
```
Comment (PiotrSikora) [inline tools/packaging/common/envoy_bootstrap.json:37]: That's a few orders of magnitude more than Envoy can handle. Is the intent here to have it "off" by default?

Comment (ramaraochavali) [reply tools/packaging/common/envoy_bootstrap.json:37]: Yes. So that it does not warn

**Review by PiotrSikora — APPROVED** [review]
Thanks!

**Pull request merged**